src/vs/workbench/contrib/notebook/browser/view/renderers/webviewMessages.ts

```diff
@@ -329,6 +330,7 @@ export interface IMarkupCellInitialization {
 	content: string;
 	offset: number;
 	visible: boolean;
+	metadata: NotebookCellMetadata;
```

Comment:
We might need this for `export interface IShowMarkupCellMessage {` too for markdown update.

Reply:
Not totally sure why `IShowMarkupCellMessage` would need a field for metadata, but I can add it there as well?